all_repos/azure_repos_api.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import json
+import urllib.request
+from typing import Any
+from typing import NamedTuple
+
+
+class Response(NamedTuple):
+    json: Any
+
+
+def req(url: str, **kwargs: Any) -> Response:
+    resp = urllib.request.urlopen(urllib.request.Request(url, **kwargs))
+    return Response(json.load(resp))
+
+
+def get_all(url: str, **kwargs: Any) -> list[dict[str, Any]]:
+    resp = req(url, **kwargs)
+    return resp.json['value']
```

Comment:
the other `get_all` functions involve pagination -- this one doesn't do any of that so yo udon't need this function